### ISSUE TITLE
Un-boosting entries and comments does not federate correctly

### ISSUE THREAD

**Issue by ghost** [MbinOrg/mbin#293]
**Describe the bug**
Un-boosting entries and comments does not federate correctly, Mbin (and probably /kbin) instances can't tell the difference and seem to assume it's adding a boost. Luckily it does not appear to duplicate boost count since there is logic in place to only allow 1 boost per `ap_id` from the user for a given entity.

**On which Mbin instance did you find the bug?**
Local dev cluster running main branch, all Mbin (and probably /kbin) instances impacted.

**To Reproduce**
1. Visit any thread or comment
2. Click on boost
3. View the same thread or comment from a different instance (that's properly federating) to verify boost was added
4. Return to the origin instance, click on boost again to remove boost
5. The origin/local instance will show the boost removed, but the entity on the federated instance stays boosted

**Expected behavior**
Removing a boost should federate as removing an upvote and downvote

**Additional context**

Serializer output for upvote/downvote:

```
App\Message\ActivityPub\Outbox\LikeMessage
App\Message\ActivityPub\Outbox\LikeMessage
  +userId: 1
  +objectId: 14
  +objectType: "App\Entity\Entry"
  +removeLike: false
}
```

Take note of the `removeLike` field

vs.

Serializer output for boosts (same regardless of adding or subtracting a boost)

```
App\Message\Notification\VoteNotificationMessage
App\Message\Notification\VoteNotificationMessage
  +subjectId: 14
  +subjectClass: "App\Entity\Entry"
}
```

**Comment by BentiGorlich** [comment]
Yeah that should result in a message of type `Undo` with the boost activity (which is just an activity of type `Announce`) as the `object`
